content/applications/marketing/events/event_tracks.rst

```diff
@@ -0,0 +1,303 @@
+============
+Event tracks
+============
+
+Odoo *Events* provides the ability to create, schedule, and manage talks, lectures, demonstrations,
+presentations, etc., which known as *Tracks* in Odoo.
+
+The Odoo *Events* application also has an option to allow event attendees to propose talks (tracks)
+for an event, which can then be approved (or disapproved).
+
+Configuration
+=============
+
+First, there are some settings that need to be enabled, in order to get the most out of event
+tracks.
+
+To do that, navigate to :menuselection:`Events app --> Configuration --> Settings`, and tick the
+checkbox beside the :guilabel:`Schedule & Tracks` setting. Doing so provides the ability to manage
+and publish an event schedule with various tracks.
+
+Also, when that setting checkbox is ticked, two additional setting options appear beneath it:
+:guilabel:`Live Broadcast` and :guilabel:`Event Gamification`.
+
+.. image:: event_tracks/track-settings.png
+   :align: center
+   :alt: The Schedule and Tracks setting in the Odoo Events app, along with corresponding options.
+
+The :guilabel:`Live Broadcast` option provides the ability to air tracks online through a *YouTube*
+integration.
+
+The :guilabel:`Event Gamification` options provides the ability to share a quiz with track attendees
+once the track is over, in order to test how much they learned.
+
+.. note::
+   With the :guilabel:`Event Gamification` setting enabled, an :guilabel:`Add Quiz` button appears
+   on track forms, providing the ability to :ref:`quickly create a quiz <events/add-quiz>` specific
+   to the topic related to that particular track.
+
+Once all desired settings have been enabled, be sure to click the :guilabel:`Save` button in the
+upper-left corner of the :guilabel:`Settings` page.
+
+Event Tracks page
+=================
+
+To access, modify, and/or create tracks for an event, first navigate to a preconfigured event, or
+:doc:`create a new one <create_events>` from the *Events* application.
+
+To do that, navigate to :menuselection:`Events app`, and either select a pre-existing event from the
+:guilabel:`Events` dashboard, or create a new one by clicking :guilabel:`New`.
+
+Once on the desired event form, click into the :guilabel:`Tracks` smart button at the top of the
+form.
+
+.. tip::
+   If that smart button is not readily available, click the :guilabel:`More` :icon:`fa-sort-desc`
+   drop-down menu to reveal hidden smart buttons. Then, click :guilabel:`Tracks` from the resulting
+   drop-down menu.
+
+Clicking that smart button reveals the :guilabel:`Event Tracks` page for that particular event,
+which presents all the tracks (both scheduled and proposed) for the event, if there are any.
```

Comment:
Nitpick– to avoid "that"
```suggestion
   If the :guilabel:`Tracks` smart button is not readily available, click the :guilabel:`More`
   :icon:`fa-sort-desc` drop-down menu to reveal hidden smart buttons. Then, click
   :guilabel:`Tracks` from the resulting drop-down menu.

Clicking the :guilabel:`Tracks` smart button reveals the :guilabel:`Event Tracks` page for that
particular event, which presents all the tracks (both scheduled and proposed) for the event, if
there are any.
```